Fixed rock-pi-s/rock-s0 Mac address duplication issue

RadxaMitchell opened the pull request.

Commit:
configs:rockpi-s/rock-s0:enable CONFIG_MISC and CONFIG_ROCKCHIP_OTP

To fix the RK3308 MAC address duplication issue.
The reason for the duplicate Mac address is because the cupid is not retrieved from efuse.
Need to turn on CONFIG_ROCKCHIP_OTP to support cpuid retrieval from efuse, and CONFIG_ROCKCHIP_OTP depends on CONFIG_MISC.

Signed-off-by: Chuang Ma <machaung@radxa.com>

RadxaYuntian commented on configs/rock-pi-s-rk3308_defconfig:
```diff
@@ -19,6 +19,8 @@ CONFIG_SPL_STACK=0x400000
 CONFIG_DEBUG_UART_BASE=0xFF0A0000
 CONFIG_DEBUG_UART_CLOCK=24000000
 CONFIG_MISC_INIT_R=y
+CONFIG_MISC=y
```
CONFIG_MISC的目的是什么？这个应该打开了吧

RadxaMitchell replied:
CONFIG_ROCKCHIP_OTP 这个宏依赖 CONFIG_MISC ，rockpi-s 和 rock-s0 的 defconfig 中 CONFIG_MISC 没有被打开

Commit:
Add e-fuse device node with otp to rk3308.dtsi

Getting the cpuid needs to be done based on the e-fuse node obtained from DTS.

Signed-off-by: Chuang Ma <machaung@radxa.com>

RadxaYuntian reviewed (approved):
`include/configs/rockchip-common.h`里面定义了`CFG_CPUID_OFFSET`为7,所以不需要额外调整CPU ID的偏移
